### PR TITLE
[Hived]: Add Feature Demo Doc

### PR DESCRIPTION
**Pull request by yqwang-ms**

### CODE REVIEW

File: subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/README.md [hunk 1]
```diff
@@ -1,24 +1,43 @@
...
+
+HiveD schedules all containers within a job in a *transactional* manner, i.e., all these containers' requirements will be granted or denied as a whole, thus avoiding partial resource allocation and starvation.
+
+### Priorities
+
+HiveD supports multiple job **priorities**. Higher-priority jobs can **[preempt](example/feature/README.md#Intra-VC-Preemption)** lower-priority jobs. HiveD also introduces **[opportunistic jobs](example/feature/README.md#Opportunistic-Job)**, i.e., jobs with lowest priority which can use other VCs' free resource when possible (without breaking the resource guarantees to other VCs).
+
+### Fault-Tolerance
+
+HiveD is **fault-tolerant**. It can candle random failures of both the scheduler itself and the resource it manages.
```
Comment (yqwang-ms) [inline subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/README.md]: >can candle [](start = 32, length = 10)

can handle

**Comment by scarlett2018** [comment]
not sure about what the "Feature Demo" is referring here, are these docs only for demo purpose?... @yqwang-ms

**Comment by yqwang-ms** [comment]
> not sure about what the "Feature Demo" is referring here, are these docs only for demo purpose?... @yqwang-ms

For now, they are just used to demonstrate our features to user.

**Comment by scarlett2018** [comment]
> > not sure about what the "Feature Demo" is referring here, are these docs only for demo purpose?... @yqwang-ms
> 
> For now, they are just used to demonstrate our features to user.

if HiveD is the default scheduler, we might make it more formal ;-).

**Comment by yqwang-ms** [comment]
> if HiveD is the default scheduler, we might make it more formal ;-).

It is formal demonstration and can be reproduced, see PR.
What do you mean by "more formal"?

**Comment by scarlett2018** [comment]
> > if HiveD is the default scheduler, we might make it more formal ;-).
> 
> It is formal demonstration and can be reproduced, see PR.
> What do you mean by "more formal"?
> 
> Or any suggestion to use better words to replace "Feature Demo"? :)

i.e. change the "example" folder to something else. minor suggestion, you can ignore this for now.

**Comment by yqwang-ms** [comment]
> > > if HiveD is the default scheduler, we might make it more formal ;-).
> > 
> > 
> > It is formal demonstration and can be reproduced, see PR.
> > What do you mean by "more formal"?
> > Or any suggestion to use better words to replace "Feature Demo"? :)
> 
> i.e. change the "example" folder to something else. minor suggestion, you can ignore this for now.

Ok, if we come up with a better name, we can change this later.

**Pull request merged**